src/test/scala/ztis/model/SparkSpec.scala

```diff
@@ -0,0 +1,27 @@
+package ztis.model
+
+import org.apache.spark.{SparkConf, SparkContext}
+import org.scalatest.{BeforeAndAfter, FlatSpec}
+
+class SparkSpec extends FlatSpec with BeforeAndAfter {
+  private val master = "local[2]"
+  private val appName = "spark-test"
+
+  protected var sc: SparkContext = _
+
+  before {
```

Comment:
I suspect we can reuse single Spark instance in different test cases - I would try to use BeforeAndAfterAll trait from scalatest